### PR TITLE
OOP school library: Add classes

### PR DESCRIPTION
**Pull request by Strangeal**
So far it has been enjoyable😊 implementing all the necessary requirements for the project I learned a lot working on this project and it was also fun, what I did in the whole project :

- Added `Person`, `Student`, and `Teacher` classes
- Added `private` and `public` methods in classes
- Added a descriptive `README` for the project

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return if @age >= 18 && parent_permission
+
+    true
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```
Comment (m05e5) [inline person.rb]: - [ optional ] Hi it's always better to follow best practices(DRY). in the `can_user_services` function, you could replace the  `if @age >= 18` with the private function you created `of_age?`

**Review by m05e5 — APPROVED** [review]
Hi @Strangeal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ your linters are all ok
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (Strangeal) [reply person.rb]: @m05e5  Thanks for the suggestion but can I use the method `of_age?` inside `can_user_services` before initializing or calling it

Comment (m05e5) [reply person.rb]: Yeah, I think. you can try and see if it works

**Pull request merged**